lib/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ etn_target(static ${PROJECT_NAME}-lib
     USES_PRIVATE
         cidr
         nutclient
+        nutclientstub
```

Comment:
Is it needed for the lib or only for test? If it's only for tests, it should not be part of the lib and not be part of the package too. If it's needed also for the runtime package, you need to upgrade the simple image to add the package inside.